sp/src/game/server/hl2/npc_attackchopper.cpp

```diff
@@ -685,7 +694,12 @@ class CNPC_AttackHelicopter : public CBaseHelicopter
 	};
 
 #ifdef HL2_EPISODIC
+#ifdef EZ2
+protected:
+	virtual void InitBoneFollowers( void );
```

Comment:
Does this need a 'private' afterwards to prevent making all these variables private?